index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./assets/css/home.css">
+    <script src="https://kit.fontawesome.com/1f634f9b31.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="set-background">
+    <header class="header">
+        <nav class="social-media" aria-label="social media icons">
+            <ul class="social-media-ul">
+                <li><a href=""><img src="./assets/images/social-media-images/facebook.png" alt=""></a></li>
+                <li><a href=""><img src="./assets/images/social-media-images/twitter.png" alt=""></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://github.com/yusufyusice">My github page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+          <img id="hamburger-img" class="hamburger-img" src="./assets/icons/menu.png" alt="hamburger mobile menu">
+         <nav class="nav-bar" aria-label="nav bar">
+            <img class="my-logo" src="./assets/icons/logo.png" alt="cc logo">
```

Comment:
- [ ] I would highly recommend wrapping your logo in hyperlink tag `<a>` as it is standard for a website logo to navigate to the home page. For more information click [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)